app/Models/NewsPost.php

```diff
@@ -27,6 +27,9 @@
  * @property string|null $tumblr_id
  * @property \Carbon\Carbon|null $updated_at
  * @property string|null $version
+ * @method static Builder default()
```

Comment:
fwiw laravel 12 has [a new method for defining scopes](https://laravel.com/docs/12.x/eloquent#local-scopes) which probably isn't compatible with this...